src/main/js/controls/Gantt/Track.js

```diff
@@ -98,9 +99,14 @@ const addTrackLabelEventHandler = (canvasSVG, labelObj) =>
                     !utils.isFunction(labelObj.onClick)
                 );
             }
-            d3.select(this).on("click", () => {
-                labelObj.onClick(labelObj.display, d3.select(this));
-            });
+            if (
+                labelObj.display.length >
```

Comment:
There is a nifty helper for this `shouldTruncateLabel`

Reply:
Fixed here https://github.com/cerner/carbon-graphs/pull/89/commits/869cb84d5e749a87f211ddc7135beec7d007283c.